UsingAuthentication.md

```diff
@@ -68,7 +68,7 @@ There are two important things to look at here: (1) the constructor which takes
 
 Let's look at the filter method first.  This class, in general, uses the standard workflow for using Rest-Assured filters (i.e. implementing the Rest-Assured Filter interface).  In this filter method (which must be implemented per the filter), you should place all the code you require in order to augment your call to have it properly authenticate.  You can import whatever external class and libraries you need, in order to add information to your call.  In this sample code, the filter checks to see if there is already an "Authorization" header in the RestAssured RequestSpec, and if there isn't, it calls a static method on an external class to retrieve a token for the header.  In your filter, you would leverage your own authentication flows.
 
-The Constructor which takes an argument, allows you to pass in service specific data at runtime in order to be able to re-use this filter for multiple services, if you require that.
+The Constructor which takes an argument allows you to pass in service-specific data at runtime in order to be able to re-use this filter for multiple services, if you require that.
```

Comment:
Actually, in this case, it looks like the comma is correct, and we're actually missing a preceeding comma after "The Constructor".  It flows better with this break  Can you add that additional comma.

Also, the hyphen here is correct.

Reply:
Added commas as described